text-input-field.js

```diff
@@ -30,6 +31,10 @@ H5P.TextInputField = (function ($) {
       requiredField: false
     }, params);
 
+    // Set the maximum length for the textarea
+    this.maxTextLength = (typeof this.params.maximumLength === 'undefined') ?
```

Comment:
Better to put this all one big line.

Reply:
Sorry. I had not noticed that you included all the new language definitions after I created the pull request. Back then, there were only three + English. I'll add the new ones today.